multinode_integration_tests/tests/verify_bill_payment_utils/utils.rs

```diff
@@ -0,0 +1,1003 @@
+    let now = SystemTime::now();
+    let cluster = match MASQNodeCluster::start() {
+        Ok(cluster) => cluster,
+        Err(e) => panic!("{}", e),
+    };
+    let blockchain_server = BlockchainServer::new("ganache-cli");
+    blockchain_server.start();
+    blockchain_server.wait_until_ready();
+    let server_url = blockchain_server.url().to_string();
+    let (event_loop_handle, http) =
+        Http::with_max_parallel(&server_url, REQUESTS_IN_PARALLEL).unwrap();
+    let web3 = Web3::new(http.clone());
+    let seed = make_seed();
+    let (contract_owner_wallet, _) =
+        make_node_wallet_and_private_key(&seed, &derivation_path(0, 0));
+    let chain = cluster.chain();
+    let contract_owner_addr = deploy_smart_contract(&contract_owner_wallet, &web3, chain);
+    let blockchain_interface =
+        Box::new(BlockchainInterfaceWeb3::new(http, event_loop_handle, chain));
+    let blockchain_params = BlockchainParams {
+        chain,
+        server_url,
+        contract_owner_addr,
+        contract_owner_wallet,
+        seed,
+    };
+    let blockchain_interfaces = BlockchainInterfaces {
+        standard_blockchain_interface: blockchain_interface,
+        web3,
+    };
+    let global_values = GlobalValues {
+        test_inputs,
+        blockchain_params,
+        blockchain_interfaces,
+        now_in_common: now,
+    };
+    assert_eq!(
+        contract_owner_addr,
+        chain.rec().contract,
+        "Either the contract has been modified or Ganache is not accurately mimicking Ethereum. \
+         Resulted contact addr {:?} doesn't much what's expected: {:?}",
+        contract_owner_addr,
+        chain.rec().contract
+    );
+
+    (cluster, global_values, blockchain_server)
+}
+
+fn make_seed() -> Seed {
+    let mnemonic = Mnemonic::from_phrase(MNEMONIC_PHRASE, Language::English).unwrap();
+    Seed::new(&mnemonic, "")
+}
+
+pub fn to_wei(gwei: u64) -> u128 {
+    gwei_to_wei(gwei)
+}
+
+fn make_db_init_config(chain: Chain) -> DbInitializationConfig {
+    DbInitializationConfig::create_or_migrate(ExternalData::new(
+        chain,
+        NeighborhoodModeLight::Standard,
+        None,
+    ))
+}
+
+fn load_contract_in_bytes() -> Vec<u8> {
+    let file_path =
+        standard_dir_for_test_input_data().join("smart_contract_for_on_blockchain_test");
+    let mut file = File::open(file_path).expect("couldn't acquire a handle to the data file");
+    let mut data = String::new();
+    file.read_to_string(&mut data).unwrap();
+    let data = data
+        .chars()
+        .filter(|char| !char.is_whitespace())
+        .collect::<String>();
+    data.from_hex::<Vec<u8>>()
+        .expect("bad contract: contains non-hexadecimal characters")
+}
+
+lazy_static! {
+    static ref GAS_PRICE: ethereum_types::U256 =
+        50_u64.try_into().expect("Gas price, internal error");
+    static ref GAS_LIMIT: ethereum_types::U256 =
+        1_000_000_u64.try_into().expect("Gas limit, internal error");
+}
```

Comment:
I'm thinking I could scan through this file with a quick peep and neaten what really catches me hard as feeling awkward about it.